### ISSUE TITLE
Focus visible pollyfill is missing from 0.18.x

### ISSUE THREAD

**Issue by gleung93** [necolas/react-native-web#2482]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

I'm in the process of upgrading from version `0.17.7` to `0.18.12` and I noticed that the data attribute `data-focusvisible-pollyfill="true"` no longer appears on elements when using the keyboard to focus them. I was using this attribute to apply specific styles only when navigating via keyboard.

Is this a regression or was it intentionally removed? If so is there another way to implement the focus visible behavior?

### Expected behavior

Elements have a `data-focusvisible-pollyfill` value when tabbing to them

### Steps to reproduce

**0.17.7 Focus visible pollyfill works: https://codesandbox.io/s/hungry-scott-kkytnj**
1. Tab to button
2. Inspect button element
3. Notice the `data-focusvisible-pollyfill="true"` attribute

**0.18.x Does not work: https://codesandbox.io/s/elegant-lake-zufnp6**
1. Tab to button
2. Inspect button element
3. No additional attribute

### Test case

https://codesandbox.io/s/hungry-scott-kkytnj

**Comment by necolas** [comment]
https://github.com/necolas/react-native-web/releases/tag/0.18.0

> The :focus-visible polyfill has been removed, as by default modern browsers no longer show focus rings for pointer interactions.

That attribute was a private implementation detail and not a public API. And the styling API doesn't give you a way to style based on attributes. If you're using another styling system and hooking into implementation details, there's no replacement for that going forward.
